### ISSUE TITLE
Avatar pieces rotate in an incorrect offset sometimes

### ISSUE THREAD

**Issue by digisomni** [vircadia/vircadia#1026]
This may be able to be solved by cachebusting the avatar. It worked for me once.

https://cdn.discordapp.com/attachments/654138603856986113/809029842875056169/unknown.png

**Comment by ctrlaltdavid** [comment]
I just saw this with the Woody avatar: eyes and mouth were somewhat disconnected from the head and floating around semi-independently as walked in selfie mode or alt-cammed around.
I was using my own dev build of, essentially, latest master. It's not happening now, though.

**Comment by HifiExperiments** [comment]
I can't reproduce this one...nothing changed about the HFM format so I don't see why it would be related to caching, but if reloading the cache fixes it...idk what's going on there.  it SOUNDS related to the changes I made, but none of them would only happen sometimes.

will keep trying

**Comment by Aitolda** [comment]
Just adding a [quick video](https://aitolda.s3.amazonaws.com/Vircadia/Bugs/2021-03-01%2018-51-25.mkv) showing the extent of it.  It's an interesting one, it's not just the rotation on attachments, but my entire avatar is moving with the camera pitch.

**Comment by ctrlaltdavid** [comment]
May have arisen with https://github.com/vircadia/vircadia/pull/1041
Or could be related to billboarding.
Only happens with avatars with multiple meshes.

**Comment by Aitolda** [comment]
@HifiExperiments

I believed I've traced it's introduction back to the second build of [Implement BillboardMode for all Entity Types](https://github.com/vircadia/vircadia/pull/961).

I can't repro it on the first build, but it's consistently reproducible from the second build on.  There are two commits in between

[billboarding for model entities](https://github.com/vircadia/vircadia/pull/961/commits/2a27fc4de2344b877920cbe8dbe3427cfe794073)
and
[optimize picking](https://github.com/vircadia/vircadia/pull/961/commits/1f71a23082bbc415a7a12ba9b5a2001a1d5dc4cf)

:(

**Comment by HifiExperiments** [comment]
yeah it’s definitely https://github.com/vircadia/vircadia/pull/961/commits/2a27fc4de2344b877920cbe8dbe3427cfe794073...likely I messed up something very minor in CauterizedModel or CauterizedMeshPartPayload

**Comment by ctrlaltdavid** [comment]
Just encountered this with the woody and noticed that when walking, the woody body only faced forward when walking in one particular direction. If walking in other directions, the woody body was rotated relative to the direction it should be facing:
- Walking forward in one particular direction the woody body is faces forwards as expected.
- Rotate your orientation by 45 degrees from original and start walking forward. The woody body is angled 45 degrees off from the expected direction.
- Rotate your orientation by a total of 90 degrees from original and start walking forward, The woody body is angled 90 degrees from expected direction, i.e., it is walking sideways.
- Rotation your orientation by a total of 180 degrees from original and start walking forward. The woody body is angled 180 degrees from expected direction, i.e., it is walking backwards.
In summary: the woody body is rotated twice the amount it should be from the "zero" direction.

Note: I don't know if the "zero" direction is relative to the original direction rezzed or if its relative to yaw = 0 relative to the world axes.

**Comment by SilverfishVR** [comment]
just had this, I'm running PR1054 but that should not matter.

I started up in selfie mode (setting from least session) and my face is off and my whole body is billboarded 😮 
![ohno](https://user-images.githubusercontent.com/53531160/110549300-25f39380-8132-11eb-97fc-01cb6a2e4562.gif)
